### PR TITLE
Header and section branch

### PR DESCRIPTION
**Pull request by Olamarx**
This portfolio project has the header and the section aspect.
I only worked on the 500px max-width media query.
The mockup was from https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1
However, I adapted it to mine.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,8 +6,67 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/7cbe454c49.js" crossorigin="anonymous"></script>
 </head>
 <body>
+    <header class="header-container">
+    <div class="sub-container">
+        <a href="#" class="logo-cont">Olamarx</a>
+      <button type="submit" class="header-font"><i class="fa fas-solid fa-bars font"></i> </button>
```
Comment (AnselemOdims) [inline index.html]: - [ ] Please wrap the hamburger button in a `nav` tag.

File: index.html [hunk 2]
```diff
@@ -6,8 +6,67 @@
...
+                    <img src="./images/Ellipse3.png" alt="Ellipse">
+                </div>
+
+            </div>
+        </div>
+<div class="porfolio-content">
+    <div class="portfolio-h1">
+        <h1>I'm Olamarx </h1>
+        <h1 class="span-portfolio-h1-1">Glad to have</h1>
+        <h1 class="span-portfolio-h1-2">you here!</h1>
```
Comment (AnselemOdims) [inline index.html]: - [ ] Kindly ensure that the `h1` is not repeated in any sections and should only be used once per page. You can use lower headings from `h2-h6`

File: index.html [hunk 3]
```diff
@@ -6,8 +6,67 @@
...
+        CONNECT WITH ME
+</p>
+
+<div class="contact-link">
+    <div class="social-link">  
+    <a href="#"><i class="fa fas-brands fa-twitter social"></i></a>
+    <a href="#"><i class="fa fas-brands fa-linkedin social"></i></a>
+    <a href="#"><i class="fa fas-brands fa-facebook social"></i></a>
+    <a href="#"><i class="fa fas-brands fa-github social"></i></a>
+</div>
```
Comment (AnselemOdims) [inline index.html]: - [ ] Please kindly ensure the list of icons is grouped with a `ul` tag, and `li` per icon; each icon has a link (`a` tag)
- [ ] Please ensure to use the list of icons specified in the original design. You can export from figma and then use in your project

File: index.html [hunk 4]
```diff
@@ -6,8 +6,67 @@
...
+                    <img src="./images/Ellipse5.png" alt="Ellipse">
+                </div>
+
+                <div class="img rigth-img">
+                    <img src="./images/Rectangle4.png" alt="Rectangle">
+                    <img src="./images/Ellipse3.png" alt="Ellipse">
+                </div>
+
+            </div>
+        </div>
```
Comment (AnselemOdims) [inline index.html]: - [ ] Please make sure the background images uses a proper background images included through the stylesheet by using the `background-image` property

File: index.html (hunk 1 above)
Comment (AnselemOdims) [inline index.html]: - [ ] Please make sure to stick to the original [figma]() design guidelines(fonts, margins, paddings, colors etc)
- [ ] Also, you should use the hamburger specified in the design. You can export it from figma and then include it in the project.

### Your Design
image

### Original Design
image

File: index.html [hunk 5]
```diff
@@ -6,8 +6,67 @@
...
+    <p class="portfolio-para">
+        I am a Web Developer! I can help you build a standard and responsive website. 
+        You can look through some of my handiworks and experience. <span class="portfolio-para-span">Do you like what you see?</span>
+   LET US CONNECT </p>
+</div>
+</div>
+
+
+<p class="connect-with-me">
+        CONNECT WITH ME
```
Comment (AnselemOdims) [inline index.html]: - [ ] Please make sure to stick to the original [figma](https://github.com/Olamarx/Microverse-Portfolio-setup/pull/1/files) design guidelines(fonts, margins, paddings, colors etc)
- [ ] Your design should look identical to the original design. At the moment, it does not. Please make adjustments to match the design

### Your design
image

### Original Design
image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @Olamarx, 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Olamarx** [comment]
I have edited the entire page. However with the same content.
save

File: index.html [hunk 6]
```diff
@@ -9,63 +9,51 @@
...
 <body>
-    <header class="header-container">
-    <div class="sub-container">
-        <a href="#" class="logo-cont">Olamarx</a>
-      <button type="submit" class="header-font"><i class="fa fas-solid fa-bars font"></i> </button> 
-    </div>
+    <header>
+        <nav class="main-container">
+        <a href="#" class="logo"> LOGO </a>
+        <a href="#" class="menu">
```
Comment (MeqdamAlqudah) [inline index.html]: - [ ] Please make sure to stick to the original Figma design guidelines(fonts, margins, paddings, colors etc)

 Also, you should use the hamburger specified in the design. You can export it from Figma and then include it in the project.
Your Design
image

Original Design
image

File: index.html [hunk 7]
```diff
@@ -9,63 +9,51 @@
...
+            </li>
+            <li>
+              <a href="#" class="social-media social-icon">
+                <img src="./images/Linkedinicon.png" alt="Linkedin"/>
+              </a>
+            </li>
+            <li>
+              <a href="#" class="social-media social-icon">
+                <img src="./images/Vector(3).png" alt="Angel"/>
+              </a>
```
Comment (MeqdamAlqudah) [inline index.html]: - [ ] Please make sure that the icons you created are in the same order as the original design 

image

File: style.css [hunk 8]
```diff
@@ -1,169 +1,112 @@
...
 
-/* End of Header */
-
-/* Main Section */
-
-.main-container {
-  padding: 10px;
-
-  /* position: absolute; */
+.main-header-content {
```
Comment (MeqdamAlqudah) [inline style.css]: - [ ] Please make sure that you are using the same background image as in [Figma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=37%3A2238)

image

File: style.css [hunk 9]
```diff
@@ -1,169 +1,112 @@
...
+.main-header-title-content {
   display: flex;
-  justify-content: space-between;
+  width: 327px;
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 55px;
+  line-height: 64px;
+  color: var(--color);
```
Comment (MeqdamAlqudah) [inline style.css]: - [ ] Please make sure to stick to the original [figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=37%3A2238) design guidelines(fonts, margins, paddings, colors etc)

- [ ] Your design should look identical to the original design. At the moment, it does not. Please make adjustments to match the design

image

image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @Olamarx , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

almost there

## Required Changes ♻️


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**